src/Reducer/UserDataReducer.js

```diff
@@ -48,6 +48,50 @@ export function UserDataReducer(state, action) {
         JSON.stringify(new_history_after_remove_all)
       );
       return { ...state, history: new_history_after_remove_all };
+    case "ADD_TO_PLAYLIST":
+      const new_playlist_after_add = action.payload;
+      localStorage.setItem(
+        "my_playlist",
+        JSON.stringify(new_playlist_after_add)
+      );
+      return { ...state, playlist: new_playlist_after_add };
+    case "ADD_VIDEO_TO_PLAYLIST":
+      const playlist_to_change = action.payload;
+
+      const new_playlist_after_SingleVideoAdd = state.playlist.map((ele) =>
```

Comment:
variable name can be short and use of camel case would be ideal.

Reply:
Done